nautobot/extras/jobs.py

```diff
@@ -979,21 +990,23 @@ def run_job(data, request, job_result_pk, commit=True, *args, **kwargs):
         job_result.completed = timezone.now()
         job_result.save()
         return False
+
     job = job_class()
```

Comment:
I think for safety we want a `try/except` around the `job_class()` init call so that an error being thrown here is still handled gracefully.